README.md

```diff
@@ -47,3 +47,9 @@ The elements provided:
 * CPR Element - Element which queries the Danish CPR register when changed.
 * CPR Name Element - This is populated with the name from the above mentioned query result.
 * CPR Address Element - This is populated with the address from the above mentioned query result.
+
+## Coding standards
+
+```shell
+./vendor/bin/phpcs --standard=Drupal --extensions=php,module,inc,install,test,profile,theme,css,info,txt,md,yml .
```

Comment:
We should exclude `vendor` here. And probably add a `phpcs.xml.dist` file a la

```xml
<?xml version="1.0"?>
<ruleset name="PHP_CodeSniffer">
  <description>The coding standard.</description>

  <file>.</file>
  <exclude-pattern>vendor/</exclude-pattern>

  <!-- Show progress of the run -->
  <arg value="p"/>

  <arg name="extensions" value="php,module,inc,install,test,profile,theme,css,info,txt,yml"/>
  <config name="drupal_core_version" value="8"/>

  <rule ref="Drupal">
    <!-- We want to be able to use "package" and "version" in our custom modules -->
    <exclude name="Drupal.InfoFiles.AutoAddedKeys.Project"/>
    <exclude name="Drupal.InfoFiles.AutoAddedKeys.Version"/>
  </rule>

  <rule ref="DrupalPractice"/>
</ruleset>
```

and then just run `./vendor/bin/phpcs --standard=phpcs.xml.dist`.